.github/workflows/release.yml

```diff
@@ -186,10 +186,18 @@ jobs:
             echo "Version ${{ github.event.inputs.version }} is a prerelease, skipping update of LAST_RELEASE"
           fi
 
+      - name: Remove generated files
+        run: |
+          # These generated files are only needed when building the final
+          # binary and should be not be present in the repository afterwards.
+          find . -name '*.generated.go' | xargs git rm
```

Comment:
Wouldn't a `-delete` argument to find achieve the same? Is `git rm` really necessary or would a `rm` do as well

Reply:
There's a small advantage in using `git rm` because it's like a `rm && git add`, so the removed file is automatically placed in staging area. Not a big difference though 😄